### DISCUSSION TITLE
Unknown function uploaded_file_mime_type('Image') (using v0.17.1)

### DISCUSSION THREAD

**Discussion by matthewlarkin** [sqlpage/SQLPage#204] [Q&A] [answered]
Hey SQLPagers,

I'm having trouble with [`sqlpage.uploaded_file_mime_type()`](https://sql.ophir.dev/functions.sql?function=uploaded_file_mime_type#function), returning `Unknown function`? Has anyone else run into this issue?

I've got the following:
```sql
set mime_type = sqlpage.uploaded_file_mime_type('Image');
select 'debug' as component, $mime_type as title;
```

which returns, `Unknown function uploaded_file_mime_type('Image')`

I'm using version 0.17.1.

🙏

**Comment by lovasoa** [comment, marked as answer]
# :grimacing: 

Looks like I added it to the documentation, wanted to implement it, and never did.
Should be fixed on `main`, you can test it on docker.

https://github.com/lovasoa/SQLpage/commit/7c38f7abc0f263a34f619061a69158c6048d6363

**Reply by matthewlarkin** [reply]
Oh super, thank you Ophir! 💪
